tests/auto_random_default/data/auto_random.t.1.sql

```diff
@@ -0,0 +1,5 @@
+/*!40103 SET TIME_ZONE='+00:00' */;
```

Comment:
can we also add an `auto_random.t.2.sql` to verify the behavior of auto-random bits?

(and `select count(distinct id >> 58) from t` should return 2.)
(no need to test it on TiDB backend due to possibility of failure as the RNG is not controllable on TiDB.)